previous-version/previous_version.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+#
+# Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+# one or more contributor license agreements. See the NOTICE file distributed
+# with this work for additional information regarding copyright ownership.
+# Licensed under the Camunda License 1.0. You may not use this file
+# except in compliance with the Camunda License 1.0.
+#
+
+# Given a current version identifies a previous version based on existing git tags
+# Params:
+#   1 - Current version. E.g. 8.8.0
+#   2 - Verbose mode (true|false)
+# Output:
+#   "$TYPE $PREV_TAG"
+#   E.g. "NORMAL 8.7.0"
+
+PREV_TAG="not found"
+VERSION=$1
+VERBOSE=$2
+
+if [ "$VERBOSE" = "true" ]; then
```

Comment:
A recommended syntax is:
```suggestion
if [[ "${VERBOSE}" = "true" ]]; then
```